src/main/java/it/gov/pagopa/cgn/portal/repository/OfflineMerchantRepository.java

```diff
@@ -0,0 +1,16 @@
+package it.gov.pagopa.cgn.portal.repository;
+
+import it.gov.pagopa.cgn.portal.model.OfflineMerchantViewEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+
+import javax.transaction.Transactional;
+
+public interface OfflineMerchantRepository extends JpaRepository<OfflineMerchantViewEntity, Long> {
+
+    @Modifying
```

Comment:
Do we need this?

Reply:
Yes. It seems that we cannot trigger correctly a "refresh" query without @Modifying.
We obtain a: "org.springframework.orm.jpa.JpaSystemException: could not extract ResultSet; nested exception is org.hibernate.exception.GenericJDBCException: could not extract ResultSet".